components/config/src/config/mod.rs

```diff
@@ -202,12 +203,18 @@ impl Config {
 
     /// Makes a url, taking into account that the base url might have a trailing slash
     pub fn make_permalink(&self, path: &str) -> String {
-        let trailing_bit =
-            if path.ends_with('/') || path.ends_with(&self.feed_filename) || path.is_empty() {
-                ""
-            } else {
-                "/"
-            };
+        let trailing_bit = if path.ends_with('/')
+            || self
+                .feed_filename
+                .as_ref()
+                .map(|ff| path.ends_with(ff))
+                .unwrap_or_else(|| path.ends_with(DEFAULT_FEED_FILENAME))
```

Comment:
I would maybe put that as a variable above so the condition is easier to read.

Reply:
Good call.  I have that one changed to keep the if statement more readable.